tests/unit/deploy/test_inferencer.py

```diff
@@ -110,7 +110,7 @@ def test_openvino_inference(task: TaskType, ckpt_path: Callable[[str], Path], da
     engine = Engine(task=task)
     export_path = ckpt_path("Padim").parent.parent
     datamodule = MVTec(root=dataset_path / "mvtec", category="dummy")
-    engine.export(
+    real_exported_path = engine.export(
```

Comment:
Looks like we don't use `export_path` from Line 111 anymore. We could remove it and rename `real_exported_path` to `export_path`?

Reply:
Not really. `export_path` points to the directory and is used in the export function. `real_exported_path` is returned by a function and indicates the .xml file

Reply:
I hope I understood you correctly

Reply:
yes, that's what I want. Perhaps, we could also remove Line 111 now? This one ?

```python 
export_path = ckpt_path("Padim").parent.parent
```